### PR TITLE
Upgrade to Rails 5.0

### PR DESCRIPTION
**Pull request by fx**
It might be a little early, but I wanted to use Rails 5 w/ Administrate.

Probably not a candidate for merging right away as it depends on a couple of unreleased gem versions (see Gemfile for notes.) But I wanted to get the process kicked off here.

This is backwards-compatible w/ Rails 4.2+, so for a merge we should swap the gemspec back to >= 4.2

No major complications to report.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+git_source(:github){ |repo_name| "https://github.com/#{repo_name}" } if respond_to?(:git_source)
```
Comment (pacso) [inline Gemfile]: The `:github` shortcut is included with bundler (as is `:gist` and `:bitbucket`), so defining a custom source for it isn't necessary.

Comment (fx) [inline Gemfile]: The default is using the git protocol, not https.

Comment (pacso) [inline Gemfile]: Fair enough.

**Comment by hayesr** [comment]
Hey, just FYI, I got things working with Rails 5 in #376

**Comment by fx** [comment]
Oh thanks for the heads up @hayesr, didn't look closely enough. Looks complete, I'll close this one.

**Pull request closed without merging**